### PR TITLE
release.yml - target_commitish REF_NAME

### PR DESCRIPTION
**Pull request by nerdCopter**
* if a github action "release" workflow is manually triggered, and a branch is selected rather than master, then the draft-release wrongly points to master rather than the branch selected.
image

* by adding `target_commitish: ${{ GITHUB.REF_NAME }}` to the `softprops/action-gh-release` step, this is rectified.
image

--
* ignore my branch name, this was for personal/testing purposes

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
It's ok to me. This action is not being used, we have other that does the build after generating the release.

**Review by haslinghuis — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**
